include/inexor/vulkan-renderer/wrapper/command_pool.hpp

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include "inexor/vulkan-renderer/wrapper/device.hpp"
```

Comment:
Can be forward declared (same goes with the device includes I think).

Reply:
I will forward declare all wrappers.